src/dataIntrefaces.ts

```diff
@@ -475,7 +534,7 @@ module powerbi.extensibility.visual {
         category: string;
         color: string;
         group: DataViewValueColumnGroup;
-        categorySelectionId: powerbi.extensibility.ISelectionId;
+        categorySelectionId: {};
```

Comment:
Why is it an object? Can we use powerbi.visuals.ISelectionId?

Reply:
We can't use. All empty interfaces were removed

Reply:
[powerbi.visuals.ISelectionId](https://github.com/Microsoft/PowerBI-visuals-tools/blob/c07894da168dac08589e7aad24b5c66eb232560d/templates/visuals/.api/v1.13.0/PowerBI-visuals.d.ts#L282) is not an empty interface